unetsocket/c/samples/txsignal.c

```diff
@@ -0,0 +1,111 @@
+///////////////////////////////////////////////////////////////////////////////
+//
+// Script to transmit a signal.
+//
+// In terminal window (an example):
+//
+// $ make samples
+// $ ./txsignal <ip_address> <signal_filename> [port]
+//
+////////////////////////////////////////////////////////////////////////////////
+
+#include <stdio.h>
+#include <stdlib.h>
+#include "../unet.h"
+#include "../unet_ext.h"
+#include <math.h>
+
+#ifndef _WIN32
+#include <unistd.h>
+#include <netdb.h>
+#include <sys/time.h>
+#endif
+
+static int error(const char *msg) {
+  printf("\n*** ERROR: %s\n\n", msg);
+  return -1;
+}
+
+int main(int argc, char *argv[]) {
+	unetsocket_t sock;
+	char id[FRAME_ID_LEN];
+  int port = 1100;
+  FILE *sigfile;
+  char *filename = NULL;
+  int ch, txbufsize = 0;
+  int rv;
+  if (argc <= 2) {
+    error("Usage : txsignal <ip_address> <signal_filename> [port] \n"
+      "ip_address: IP address of the transmitter modem. \n"
+      "port: port number of transmitter modem. \n"
+      "A usage example: \n"
```

Comment:
The description doesn't mention what `signal_filename ` is as an argument.

Reply:
Fixed.